resources/css/utilities.css

```diff
@@ -34,3 +34,8 @@
 span.smalldate {
   display: inline-block;
 }
+
+/* @see https://alpinejs.dev/directives/cloak */
+[x-cloak] {
+  display: none !important;
+}
```

Comment:
I could've sworn we added this to our CSS at one point, but I can't find any reference to it anymore. Either way, this is required to avoid a flash of unstyled content from the action menu in the progress component (where reset progress lives).